gitlab/resource_gitlab_project_cluster_test.go

```diff
@@ -85,7 +85,7 @@ func TestAccGitlabProjectCluster_import(t *testing.T) {
 				ResourceName:            "gitlab_project_cluster.foo",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"enabled", "kubernetes_token"},
+				ImportStateVerifyIgnore: []string{"enabled", "kubernetes_token", "managed"},
```

Comment:
how hard is it to add a test that creates a cluster managed and one unmanaged, even if we can not read the status?

Reply:
ok I added a test but I'm not sure it really tests much